src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -466,13 +510,13 @@ private class SamRecordWriter(prefix: PathPrefix,
                               val umiTag: String,
                               val numSamples: Int) extends DemuxWriter {
   val order: Option[SamOrder] = if (header.getSortOrder == SortOrder.unsorted) None else SamOrder(header)
-  private val writer = SamWriter(PathUtil.pathTo(s"${prefix}.bam"), header, sort=order,
+  private val writer = SamWriter(PathUtil.pathTo(s"$prefix.bam"), header, sort=order,
```

Comment:
Any reason to make this change?  I tend to think it's clearer to keep the braces unless the next character is a space.